Tests/Reqnroll.VisualStudio.Specs/Features/Editor/Commands/DefineStepsCommand.feature

```diff
@@ -127,3 +127,39 @@ Scenario: DefineSteps command abides by reqnroll.json configuration for regex sk
 			| type  | expression                        |
 			| Given | the client added (.*) pcs to the basket |
 
+Scenario: DefineSteps command properly escapes empty brackets when using Cucumber expressions
+	Given there is a Reqnroll project scope
+	And the following feature file in the editor
+		"""
+		Feature: Scenario using empty brackets
+		
+		Scenario: Client has a simple basket
+			When I call the integer.ToString() method
+		"""
+	And the project is built and the initial binding discovery is performed
+	When I invoke the "Define Steps" command
+	Then the define steps dialog should be opened with the following step definition skeletons
+		| type | expression                                 |
+		| When | I call the integer\\.ToString\\(\\) method |
```

Comment:
The proper escaped version of `I call the integer.ToString() method` would be `I call the integer.ToString\() method` (the `.` should not be escaped). See above.

Maybe we can find a better example, so that it shows all escaped characters, like `When I use (parenthesis), {curly braces} and/or \ backslash`, where the escaped should be `When I use \(parenthesis), \{curly braces} and\/or \\ backslash`

Reply:
Is the Regex snippet provider behaving correctly as-is? It does escape the period, which if I'm not mistaken is required because it could otherwise be mistaken for a Regex pattern character. Correct?

Reply:
Yes, correct.